CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Unreleased
 * Runtime now adds `system: true` to all the query parameters that it sets
 * More useful error messages for assertion failures in legacy `tests`
+* Digest auth now does not retry when all the required auth parameters were already present in the first request sent
```

Comment:
`Digest auth does not attempt retries for invalid credentials/configuration. It will continue to retry for missing configuration.`